src/solvers/dgmulti/types.jl

```diff
@@ -66,6 +66,21 @@ function DGMulti(; polydeg=nothing,
           polydeg=polydeg, kwargs...)
 end
 
+# dispatchable constructor for DGMulti using a TensorProductWedge
+function DGMulti(element_type::Wedge, 
+                 approximation_type, 
+                 volume_integral,
+                 surface_integral;
+                 polydeg::Tuple,
+                 kwargs...)
+  
+  factor_a = RefElemData(Tri(), approximation_type, polydeg[1]; kwargs...)
+  factor_b = RefElemData(Line(), approximation_type, polydeg[2]; kwargs...)
```

Comment:
Since the `Tri` and `Line` `RefElemData` constructors expect different keywords, I don't think this will work in general. 

We can leave it for now, but I would add a comment noting this.

Reply:
(or open an issue noting that `kwargs` won't work as expected for `DGMulti` solvers which use `TensorProductWedge` approximations).

Reply:
also, should we specialize `approximation_type::TensorProductWedge`?

Reply:
I don't think so, because a TensorProductWedge is a `RefElemData` and not an approximation type. I think it would make more sense to add TensorProductWedges as an element type and to spezialize for that.